src/nora/components/NoraCheckboxInput/NoraCheckboxInput.md

```diff
@@ -23,3 +23,49 @@ that accounts for the generally smaller font sizes we use in Nora.
   </a>
 </NoraCheckboxInput>
 ```
+
+Example with one checkbox setting the value of another
+
+```jsx
+import React, { useState } from 'react'
+
+function NoraCheckboxExample() {
```

Comment:
FYI @roblevintennis this was the example that I was unable to get working because there's no way to set the current checkbox value (and `currentValue` in `CheckboxInput` is the same as `initialValue`, it doesn't actually set the current value).

Reply:
Thanks for creating the test case Steph--I was able to work off it until I got the desired results for our use case. Working in isolation like this (outside of the main application) is probably 3x efficiency, and I'm glad we did it here :) We should continue to leverage this approach for EDS specific bugs / features / etc.